Services/RequestEntityService.php

```diff
@@ -124,6 +125,14 @@ private function _createRequestEntity(array $data, Request $request, string $req
 
     $requestEntity->setRequest($request);
 
+    if ($requestEntity instanceof PostDeserializeValidationInterface) {
+      $this->validator->validate(
```

Comment:
store this output in a variable, check for violations and throw error

Reply:
Wow